lib/Hakyll/Web/Pandoc/Biblio.hs

```diff
@@ -84,6 +85,14 @@ readPandocBiblio :: ReaderOptions
                  -> (Item String)
                  -> Compiler (Item Pandoc)
 readPandocBiblio ropt csl biblio item = do
+  pandoc <- readPandocWith ropt item
+  processPandocBiblio csl biblio pandoc
+
+processPandocBiblio :: Item CSL
```

Comment:
Can you please add the newlines and dashes that precede each function? It's better to keep the style consistent :)

Also, please add this function to the documentation at the top of the file.

Reply:
Thanks for the quick reply!

Done.